commit abb9063da6380f316c90801456af227f24e8521b
Author: John Levon <john.levon@joyent.com>
Date:   2019-09-19T13:52:23+00:00 (3 weeks ago)
    
    OS-7993 build_live bi_gen_version_files missing a pfexec
    Reviewed by: Alex Wilson <alex@cooperi.net>
    Approved by: Tim Foster <tim.foster@joyent.com>

diff --git a/tools/build_live b/tools/build_live
index 103d8a98..ed5528d4 100755
--- a/tools/build_live
+++ b/tools/build_live
@@ -428,7 +428,7 @@ function bi_gen_version_files
 	    fail "failed to copy gitstatus"
 
 	if ! cmp "$gitstatus" "$bvfile" >/dev/null 2>&1; then
-		cp "$gitstatus" "$bvfile" ||
+		pfexec cp "$gitstatus" "$bvfile" ||
 		    fail "could not copy build version"
 	fi
 
